### ISSUE TITLE
target.schema should return the schema for the environment being evaluated

### ISSUE THREAD

**Issue by crericha** [TobikoData/sqlmesh#998]
dbt users expect target.schema to point to the schema for their active environment (aka target). sqlmesh should match that behavior to support run-time adapter queries that dynamically fetch tables.

**Comment by tobymao** [comment]
@crericha is this still an issue?

**Comment by crericha** [comment]
Yes, still an issue. `target.schema` will point to `prod`. It's unclear to me what `schema` should be provided as the snapshot may be shared by multiple environments. We'll need to talk through this one.

**Comment by StuffbyYuki** [comment]
@crericha Has this issue ever been resolved?

**Comment by StuffbyYuki** [comment]
Closing it for now - didn't get a response from the issue owner
